common/requirements.txt

```diff
@@ -0,0 +1,21 @@
+#
+# This file is autogenerated by pip-compile
+# To update, run:
+#
+#    pip-compile --output-file requirements.txt requirements.in
+#
+boto3==1.4.4
```

Comment:
Celery needs boto2 but any code I write that uses boto will use version 3. That way once Celery upgrades to boto3 we're already on it and can just remove the dependence on boto2 without changing anything except a celery version.